addons/payment_razorpay/views/payment_form_templates.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="payment_razorpay.token_form" inherit_id="payment.token_form">
+        <xpath expr="//t[@t-if='is_test']" position="before">
+            <t t-set="razorpay_limit_exceed_warning" t-value="token_sudo._get_razorpay_limit_exceed_warning(amount)"/>
+        </xpath>
+        <xpath expr="//t[@t-if='is_test']" position="inside">
+            <t t-set="title" t-if="razorpay_limit_exceed_warning and not is_test" t-value="razorpay_limit_exceed_warning"/>
+            <t t-set="title" t-elif="razorpay_limit_exceed_warning and is_test" t-value="'%s + %s' % (title, razorpay_limit_exceed_warning)"/>
+        </xpath>
+        <xpath expr="//t[@t-if='is_test']" position="attributes">
+            <attribute name="t-if" add="or razorpay_limit_exceed_warning" separator=" "/>
+        </xpath>
```

Comment:
Hm, maybe modify the attribute before adding something inside? Both are fine, but it seems more logical to me, when reading the code, to first change the condition to have that `t`, then add something inside it

Reply:
Done! Thank you :slightly_smiling_face: